frontend/packages/dev-console/src/components/pipelineruns/detail-page-tabs/TaskRuns.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import TaskRunsListPage from '../../taskruns/list-page/TaskRunsListPage';
+
+const TaskRuns: React.FC = (props) => (
+  <TaskRunsListPage
+    showTitle={false}
+    selector={{ 'tekton.dev/pipelineRun': _.get(props, 'obj.metadata.name') }}
```

Comment:
we could replace the lodash _.get with optional chaining and remove the lodash import. why to import for this simple usage, WDYT?

Reply:
Done